manifests/complete-postgres-manifest.yaml

```diff
@@ -70,6 +70,15 @@ spec:
   # run periodic backups with k8s cron jobs
   # enableLogicalBackup: true
   # logicalBackupSchedule: "30 00 * * *"
+  #
+  # # standby cluster example.
+  # # See https://patroni.readthedocs.io/en/latest/replica_bootstrap.html#standby-cluster.
+  #
+  # standby_cluster:
+  #   host: master.hostname
+  #   port: "5432"
+  #   primary_slot_name: dmz2
```

Comment:
Can you move this part behind `maximum_lag_on_failover`. Only then it's under the `patroni` key. You can remove the black lines and maybe also `# standby cluster example.`